### ISSUE TITLE
Defined variables with value None are not reflected in radio buttons

### ISSUE THREAD

**Issue by staffanm** [jhpyle/docassemble#450]
Given the following interview:

```yaml
code: |
  ynm = None
  tristate = None
  thing = "X"
---
question: What is the state of ${thing}?
fields:
  - Yes-no-maybe variable: ynm
    datatype: yesnomaybe
  - Tri-state variable: tristate
    input type: radio
    choices:
      - On: true
      - Off: false
      - Broken: null
continue button field: done
---
mandatory: true
question: Done
subquestion: |
   % if done:
   ynm is ${ynm}, tristate is ${tristate}
   % endif

```
I would expect that the question "What is the state of X" would show "I don't know" and "Broken" for the two fields respectively. Instead these fields are blank as if the variable had not yet been defined. If I change the initial code block to set the variables to True (or False), this is reflected in the fields on the  "What is the state of X" question. 

Is it possible to have variables that are defined and have the value None rendered for either yesnomaybe or radio fields?

**Comment by jhpyle** [comment]
Thanks.  Sorry for the delay.  I believe I have fixed this in 1.2.100.

**Comment by staffanm** [comment]
Confirm that this is fixed, closing
